### ISSUE TITLE
need more documentation

### ISSUE THREAD

**Issue by dmrzh** [brix-cms/brix-cms#217]
Need more documentation how to embedd brix-cms into wicket application

**Comment by kbachl** [comment]
Hello @dmrzh ,

yes, we need more documentation, however, this will be earliest at brix 8 release that is quite a bit away at the time. You can get a basic overview here in the wiki https://github.com/brix-cms/brix-cms/wiki and to see how to add it to wicket application you might want to check out the brix-demo src here:
https://github.com/brix-cms/brix-cms/tree/master/brix-demo/src/main/java/org/brixcms/demo/web

especially the AbstractWicketApplication as well as the WicketApplication classes. 

Basically in the  init method you have to add brix PrefixUriMapper; It may be mixed with any wicket pages there as well, meaning you just add the pages like before for them to resolve or in case of any existing wicket app you would add the uri mapper there. Hope this helps you to get started.
